### ISSUE TITLE
Continuous reading not stop when using new configurations and vocalizer voices

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3540]
**Reported by ManuelRamos on 2013-09-21 23:37**
Dear Developers, NVDA is already most powerfull software for blind. And free! Thanks! A wqy to get independence!

I am using NVDA - last "next" version available - and I am having this problem.

I am using vocalizer voice Joana to read menus and things like that; and vocalizer voice Joana, more powerfull, to read text. For example to read word documents, other processor and webpages when loaded.

But, whenm opening a word document or a webpage, voice swiches, as expected; however, the text being read does not stop by pressing escape, control or even alt (start button).

In other words, text sotops being read, but after a long time, perheaps a minute or at least 30 seconds. I have not tested yet if text stops only in next paragraph.

Should I desable "say all" inin configuration triggers?

Thanks in advance (I am using Win7 64; winword 2010; firefox, last version available)

**Comment by ehollig** [comment]
CC @ragb, could you respond to @jcsteh in https://github.com/nvaccess/nvda/issues/3540#issuecomment-155309700?

**Comment by ruifontes** [comment]
Hello!

Yes, can be closed.

Rui Fontes


-----Mensagem Original----- 
De: Ethan Holliger
Data: 13 de agosto de 2017 03:41
Para: nvaccess/nvda
Cc: Subscribed
Assunto: Re: [nvaccess/nvda] Continuous reading not stop when using new 
configurations and vocalizer voices (#3540)



CC @ragb, could you respond to @jcsteh in #3540 (comment)?

**Comment by ehollig** [comment]
Closing as requested in https://github.com/nvaccess/nvda/issues/3540#issuecomment-322248209
